src/main/java/seedu/eylah/expensesplitter/model/person/UniquePersonList.java

```diff
@@ -167,4 +167,10 @@ public Person getPerson(Person person) {
         int index = internalList.indexOf(person);
         return internalList.get(index);
     }
+
+
+    public Person getPersonUsingIndex(int indexOfPerson) {
+        Person p = internalList.get(indexOfPerson); // I minused 1 when i parsed so dont minus again
```

Comment:
Can add a more formal javadoc comment on the starting index of the list (i.e. zero index or one index) so users can count correctly,

Reply:
Roger